packages/file_selector/file_selector_web/example/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ dependencies:
     sdk: flutter
 
 dev_dependencies:
-  build_runner: ^1.10.0
+  build_runner: ^2.0.0
```

Comment:
This one can probably be removed, I don't see any need for build_runner in this package (it maybe used mockito5 in the past)